### ISSUE TITLE
2D particle system

### ISSUE THREAD

**Issue by Gigabait** [Facepunch/garrysmod-requests#1570]
Why gmod not have 2D particle system in 2D context ? Like in LOVE.
Yes you can create your own particle system with lua , but if it is by default it will be much better.

**Comment by robotboy655** [comment]
Because GMod is a 3D game that runs on a 3D game engine, and Love2D is a 2D game engine. There's simply no reason to have that in either the Source Engine or in Garry's Mod.

**Comment by Gigabait** [comment]
@robotboy655 Yes Gmod is 3D game but also he using 2D Context, Like HUDPaint. Anyway its posible adding but why it not added by default ? It very helpfure for developers with UI.
